spec/components/text_sentinel_spec.rb

```diff
@@ -49,7 +49,7 @@
     [ 'evil trout is evil',
       "去年十社會警告",
       "P.S. Пробирочка очень толковая и весьма умная, так что не обнимайтесь.",
-      "LOOK: 去年十社會警告"
+      "Look: 去年十社會警告"
```

Comment:
I had to change this.

People using ideograms are already enabling upcase posts on the settings.